imap_processing/glows/l1b/glows_l1b_data.py

```diff
@@ -272,7 +273,38 @@ def __post_init__(
         pulse_test_in_progress: np.double,
         memory_error_detected: np.double,
     ):
-        """Generate the L1B data for direct events using the inputs from InitVar."""
+        """
+        Generate the L1B data for direct events using the inputs from InitVar.
+
+        Parameters
+        ----------
+        direct_events : np.ndarray
+            Direct events.
+        glows_ssclk_last_pps : int
+            No idea todo check.
+        catbed_heater_active : np.double
+            No idea todo check.
+        spin_period_valid : np.double
+            Valid spin period.
+        spin_phase_at_next_pps_valid : np.double
+            Spin phase.
+        spin_period_source : np.double
+            Spin period source.
+        glows_time_on_pps_valid : np.double
+            Glows time.
+        time_status_valid : np.double
+            Time status.
+        housekeeping_valid : np.double
+            Housekeeping.
+        is_pps_autogenerated : np.double
+            No idea todo check.
+        hv_test_in_progress : np.double
+            Hv test.
+        pulse_test_in_progress : np.double
+            Pulse test.
+        memory_error_detected : np.double
+            Memory error detector.
```

Comment:
```suggestion
            Flag indicating if a memory error is detected.
```

Sorry! I totally missed all these docs!